### PR TITLE
Removed isset() micro optimization

### PR DESCRIPTION
**Pull request by germanow**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  |

### CODE REVIEW

File: src/ArrayHelper.php [hunk 1]
```diff
@@ -196,7 +199,7 @@ public static function getValue($array, $key, $default = null)
         }
 
         if (is_array($array)) {
-            return (isset($array[$key]) || array_key_exists($key, $array)) ? $array[$key] : $default;
+            return array_key_exists($key, $array) ? $array[$key] : $default;
```
Comment (xepozz) [inline src/ArrayHelper.php]:
```suggestion
            return $array[$key] ?? $default;
```

File: src/ArrayHelper.php [hunk 2]
```diff
@@ -180,7 +180,10 @@ public static function getValue($array, $key, $default = null)
             $key = $lastKey;
         }
 
-        if (is_array($array) && (isset($array[$key]) || array_key_exists($key, $array))) {
+        if (is_float($key)) {
```
Comment (xepozz) [inline src/ArrayHelper.php]: Why did you add it?

Comment (germanow) [reply src/ArrayHelper.php]: https://github.com/yiisoft/yii2/pull/11549
